src/pages/setup/setProfile/SetJobPage.tsx

```
@@ -28,6 +28,7 @@ function SetJobPage() {
         const updatedUserInfo = { ...userInfo, category: selected };
 
         setUserInfo(updatedUserInfo);
+        debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)